features/step_definitions/document_steps.rb

```diff
@@ -223,11 +223,11 @@
   assert page.has_no_content?(edition.title)
 end
 
-Then(/^the publication should be related to "([^"]*)" and "([^"]*)" policies$/) do |related_policy_1, related_policy_2|
-  policies_titles = Publication.last.policies.map(&:title)
-
-  assert policies_titles.include?(related_policy_1)
-  assert policies_titles.include?(related_policy_2)
+Then(/^(#{THE_DOCUMENT}) should be visible to the public$/) do |edition|
+  public_edition_path = public_path_for(edition)
+  stub_content_item_from_content_store_for(public_edition_path)
+  visit_public_index_for(edition)
+  assert page.has_css?(record_css_selector(edition), text: edition.title)
```

Comment:
and this test

Reply:
This was also removed in 79e3bf6eb4bac18c40e186f5edd3d718bb0e969e, and came back from a bad rebase. I'll fix this as well.